test/extended/builds/cluster_config.go

```diff
@@ -22,6 +22,7 @@ var _ = g.Describe("[Feature:Builds][Serial][Slow] alter builds via cluster conf
 	)
 
 	g.Context("", func() {
+		g.Skip("TODO: disabled due to https://bugzilla.redhat.com/show_bug.cgi?id=1685185")
```

Comment:
in my experience the g.Skip needs to be incide the g.It() block (meaning you need to put it in each of the It() blocks).

If you understand ginkgo contexts better than me and are sure this is right, let me know, but i ran into issues where suddenly all tests were being skipped when i did stuff like this.

Reply:
also I thought you were just going to disable+reenable the MCO?

Reply:
That was a much bigger task than I originally thought (multiple operators, got push back from runtimes team). Simpler to skip & re enable once the apiserver issue is resolved

Reply:
ok.  Check the output from an e2e-aws and e2e-aws-builds run to make sure the successful test output still looks correct (ie this change didn't skip things we didn't intend to skip) and let me know... if that looks ok, it's lgtm.

Reply:
fixed the skip. e2e-aws-builds passed with three tests in question skipped. e2e-aws flaked on image build.